### PR TITLE
add getSourceLines and getSource to Block API

### PR DESCRIPTION
**Pull request by mojavelinux**

### CODE REVIEW

**Comment by mojavelinux** [comment]
I decided to use the API getSourceLines() instead of getLines() since it's a more accurate name. I may add that alias to core as well.

**Comment by mojavelinux** [comment]
It's also consistent with the method name on Document.

File: spec/node/asciidoctor.spec.js [hunk 1]
```diff
@@ -141,6 +141,8 @@ describe('Node.js', function () {
       expect(blocks[1].getBlocks()[1].getId()).toBe('blockid');
       expect(blocks[1].getBlocks()[1].getStyle()).toBe('quote');
       expect(blocks[1].getBlocks()[1].getAttribute('attribution')).toBe('Abraham Lincoln');
+      expect(blocks[1].getBlocks()[1].getSourceLines()).toEqual(['This is a quote.', 'It has a title, id, and attribution.'])
```
Comment (Mogztter) [inline spec/node/asciidoctor.spec.js]: Missing semicolon 😉 
At some point we may switch to https://standardjs.com/ and remove semicolons but in the meantime...

File: spec/node/asciidoctor.spec.js [hunk 2]
```diff
@@ -141,6 +141,8 @@ describe('Node.js', function () {
...
+      expect(blocks[1].getBlocks()[1].getSource()).toBe('This is a quote.\nIt has a title, id, and attribution.')
```
Comment (Mogztter) [inline spec/node/asciidoctor.spec.js]: Missing semicolon

File: spec/node/asciidoctor.spec.js (hunk 1 above)
Comment (Mogztter) [inline spec/node/asciidoctor.spec.js]: Missing semicolon

**Comment by mojavelinux** [comment]
Perhaps you could add the linter as part of the test task. I always forgot to run it because it's not linked.

**Comment by Mogztter** [comment]
> Perhaps you could add the linter as part of the test task. I always forgot to run it because it's not linked.

Yes that's an excellent idea. It happened to me too!

**Comment by mojavelinux** [comment]
:+1:

**Pull request merged**